plugins/modules/zos_tso_command.py

```diff
@@ -175,19 +200,24 @@ def run_module():
         module.fail_json(msg=repr(e), **result)
 
     commands = parsed_args.get("commands")
+    max_rc = parsed_args.get("max_rc")
+    if max_rc is None:
+        max_rc = 0
 
     try:
-        result["output"] = run_tso_command(commands, module)
+        result["output"] = run_tso_command(commands, module, max_rc)
+        result["max_rc"] = max_rc
         for cmd in result.get("output"):
-            if cmd.get("rc") != 0:
+            if cmd.get("rc") > max_rc:
```

Comment:
I think I could do that, but it would lead to a very long string for the fail.  I'll put the code in to try it out.  (If someone sends in 20 lines, we could be returning 20 lines because of 1 failure.). It should work if I make a new string, and put the failure based on a flag, and move the failure outside the loop so it still completes.

Reply:
I'm pushing this change up, but need to figure out some decent testing for it first.

Reply:
Valid point, but not sure how else we could or even should address this, maybe its best to handle it in a discussion over webex. I have some pretty long lines coming back from zos_job_submit and while they are not 20 they certainly are nice to have a clearer understanding of whats going on. 

I just worry someone sends over 5 commands, and they were meant to be executed in order and they don't know which one failed to know where to pick back up, as I said, we probably need a bit of an interface overhaul on input and ouput of the module to better address this , so  i will make a ticket for this soon. But for now lets figure out what is needed at this point.

Reply:
This calls for a test... I'm beginning to think this may be resolved by TSO itself... wouldn't it stop on its first error, so we wouldn't have additional output?  Meaning the first serious error would stop the process?  Also, would that impact if we can have a max_rc on multi command output.  Let's grab a few minutes for this on Monday, and figure this out.

Reply:
rc is actual rc
origrc/orig_rc have been removed
loop STOPS at first occurrence of rc > max_rc
listds expanded to use full form of command...in example and in test_func case
and it just passed build:

![image](https://user-images.githubusercontent.com/39140439/230979286-46a5fa75-f33a-46de-ae06-48a113a935fe.png)

Only concern is the string-builder isn't efficient (I add to strings instead of making a list and joining.  I may rerun later.

Reply:
Corrected the string builder, so it makes a list, then merges them when done if needed.  Rebuilt.

![image](https://user-images.githubusercontent.com/39140439/230987461-db0a5d38-98eb-423c-b3ec-04019343330e.png)